.gitignore

```diff
@@ -34,6 +34,3 @@ yarn-error.log*
 
 # vercel
 .vercel
-
-# idea
-.idea
```

Comment:
https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore die zu ignorierenden Dateien sind in .idea/.gitignore